bonfire/utils.py

```diff
@@ -175,7 +190,8 @@ def _get_ref(self, get_ref_func):
                 else:
                     alts = ", ".join(self._alternate_refs[self.ref])
                     raise Exception(
-                        f"failed to fetch git ref {self.ref} or any of its alternates: {alts}"
+                        f"failed to fetch git ref '{self.ref}' or any of its alternates: '{alts}' "
+                        f"request failed with '{response.text}'"
```

Comment:
If we are going to include `response.tex`t can we also list `response.status_code`